tests/wrappers/ReactWrapper-test.js

```diff
@@ -191,5 +192,41 @@ describe('ReactInVue', () => {
         )
       )
     })
+
+    it('works with no children', () => {
+      const spy = jest.spyOn(console, 'error')
+
+      makeVueInstanceWithReactComponent(ChildlessComponent)
+      expect(document.body.innerHTML).toBe(
+        '<div><input><div><div><span>Message for React</span><button></button></div></div></div>'
+      )
+      expect(spy.mock.calls.length).toBe(0)
+
+      spy.mockReset()
+      spy.mockRestore()
+    })
+
+    it('raises underlying prop type errors with children in childless component', () => {
```

Comment:
@akxcv this test will fail on travis as `NODE_ENV=production` is set in travis.yml, and prop types are no-op'ed in production builds. It does mean the previous test is not testing things.

Let me know how you'd like to proceed.

Reply:
Yeah, I don't think `NODE_ENV` should be `production` in Travis tests. We should remove that setting.